pkg/apis/app/v1alpha1/kogitoapp_types.go

```diff
@@ -130,6 +141,8 @@ type KogitoAppBuildObject struct {
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
 	// Internal Maven Mirror to be used during source-to-image builds to considerably increase build speed
 	MavenMirrorURL string `json:"mavenMirrorURL,omitempty"`
+	// Internal usage only, will be set by CLI when user create a kogito service from a Kogito resource, e.g. dmn/drl/bpmn standalone files
+	HasStandaloneKogitoResource bool `json:"hasStandaloneKogitoResource,omitempty"`
```

Comment:
I don't think we should expose an "internal" use only attribute in a public interface. We need to figure another way to have this feature in the CR.

Reply:
@xiezhang7 see here.